src/main/java/com/singh/astha/notification/service/controllers/NotificationController.java

```diff
@@ -0,0 +1,37 @@
+package com.singh.astha.notification.service.controllers;
+
+import com.singh.astha.notification.service.dtos.kafka.NotificationRequest;
```

Comment:
This is from kafka package. Let's not blur the boundary here.

Reply:
fixed